docs/architecture/psa-migration/outcome-analysis.sh

```diff
@@ -34,7 +34,7 @@ record() {
     export MBEDTLS_TEST_OUTCOME_FILE="$PWD/outcome-$1.csv"
     rm -f $MBEDTLS_TEST_OUTCOME_FILE
 
-    make check
+    make -j check
```

Comment:
Please don't use `make -j` in CI scripts. That does unbridled parallelism which creates load spikes and seems to decrease CI performance overall. Even where load isn't a problem and the sole goal is processing time for this single build, increasing parallelism has diminishing returns: `-j2` is significantly faster than single threading, `-jNUMBER_OF_PROCESSORS` is a little faster than `-j2`, going beyond that is pretty much no win.

We set `MAKEFLAGS` to a reasonable `-j` in our CI environments, so it's best to call `make` without an explicit `-j` in scripts.

Reply:
Ok, understood. Thanks for the explanation ;)